src/components/PreviewComment/index.js

```diff
@@ -108,7 +108,7 @@ export const styles = StyleSheet.create({
   },
   container: (totalComment) => ({
     borderLeftWidth: 1,
-    marginHorizontal: SIZES.base,
+    marginHorizontal: SIZES.base - 1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `SIZES.base` to `SIZES.base - 1` for the `marginHorizontal` attribute might cause visual inconsistencies across different screens or devices, especially if `SIZES.base` is used as a standard margin size throughout the application. If this change is necessary for a specific use case, consider creating a new constant that represents this smaller size and use it consistently for similar cases.
```diff
-    marginHorizontal: SIZES.base - 1,
+    marginHorizontal: SIZES.smallerBase,
```
Please ensure to define `SIZES.smallerBase` in your constants file as `SIZES.base - 1`. This will improve maintainability and consistency of your code.


<!-- This is an auto-generated comment by OSS CodeRabbit -->